packages/buttons/src/styled/StyledButton.ts

```diff
@@ -337,17 +353,17 @@ export const StyledButton = styled.button.attrs<IButtonProps>(props => ({
     padding: 0;
   }
 
+  &:hover {
+    text-decoration: ${props => (props.isLink ? 'underline' : 'none')}; /* [2] */
+  }
+
```

Comment:
While this shuffle honors LVHFA, it represents a change in CSS logic and the text is no longer showing underline on hover when focused.